app/scripts/directives/bindService.js

```diff
@@ -244,6 +279,9 @@
       };
 
       var serviceClass = BindingService.getServiceClassForInstance(svcToBind, ctrl.serviceClasses);
```

Comment:
**NOTE**: The serviceClass is set in the function `loadMobileServiceClass` as this returns undefined unless [this constant value](https://github.com/aerogear/origin-web-console/blob/aerogear-mcp/app/scripts/constants.js#L106) is set to `True`. (`ctrl.serviceClasses` is set in `loadApplications` which does not get called)